src/generic/FreeAssociativeAlgebra.jl

```diff
@@ -133,12 +133,17 @@ function (a::FreeAssociativeAlgebra{T})(b::T) where T
     return FreeAssociativeAlgebraElem{T}(a, T[b], [Int[]], 1)
 end
 
-function (a::FreeAssociativeAlgebra{T})(b::Integer) where T
+function (a::FreeAssociativeAlgebra{T})(b::Union{Integer, Rational, AbstractFloat}) where T
     iszero(b) && return zero(a)
```

Comment:
This needs to be adjusted. One probably does not want `A(0.0)` to work if `A` is an algebra over `GF(2)`.

Reply:
Won't `R(b)` then throw an exception a few lines later?

Reply:
no, in the particular case @thofma explains, the `iszero(b)` triggers already. The fix (pushed in a minute or so) works around the issue by checking `iszero(R(b))` instead, which fails in thofma's example